lib/connection.js

```diff
@@ -11,13 +11,52 @@ const isBase64 = (payload) => new RegExp(base64Regex, 'gi').test(payload);
 const base64Decode = (message) =>
```

Comment:
all help func move to 'util/message.js' ?

Reply:
I'll move these to files in utils.